plugins/vendor/zoom/ZoomDropFolderPlugin/batch/processors/zoomMeetingProcessor.php

```diff
@@ -7,7 +7,17 @@ class zoomMeetingProcessor extends zoomRecordingProcessor
 {
 	protected function getAdditionalUsersFromZoom($recordingId)
 	{
-		return $this->zoomClient->retrieveMeetingParticipant($recordingId);
+		return $this->zoomClient->retrieveReportMeetingParticipant($recordingId);
+	}
+
+	protected function getAlternativeHostsData($recordingId)
+	{
+		return $this->zoomClient->retrieveMeeting($recordingId);
```

Comment:
it's a bit weird to have a function named getAlternativeHostsData which returns a meeting, and then checks that it has hosts data on zoomRecordingProcessor.php:362 and extracts the data there. I would expect the function to return the alternative hosts and not the entire meeting

Reply:
changed